README.md

```diff
@@ -98,7 +98,17 @@ As a contributor, you can also point to your own fork containing the `tldr.zip`
 
 ```js
 {
-  "repository" : "http://myrepo/assets/tldr.zip",
+  "repository" : "http://myrepo/assets/tldr.zip"
```

Comment:
A bit out of context, but I noticed that there's a space in front of the colon. Is this correct? The other examples don'( have it.